CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
```

Comment:
❓ Can we simplify this to match [this file](https://github.com/dotnet/corefx/blob/master/CODE_OF_CONDUCT.md) so we can avoid dealing with updating this over time?

Reply:
I don't think so. This project isn't part of the .NET Foundation. That document gives conduct@dotnetfoundation.org as a contact email address.

Reply:
What about using that content, but with the following footnote?

> Please note that the contact email for the maintainers of this project is _____.

Reply:
That could work.

Reply:
As you likely saw, I opted to keep the full document for now and include a reference to the fact that it's the same document the .NET Foundation uses.